yt/data_objects/construction_data_containers.py

```diff
@@ -2481,13 +2488,35 @@ def export_ply(
             elif sample_type == "vertex" and color_field not in self.vertex_samples:
                 self.get_data(color_field, sample_type, no_ghost=no_ghost)
         self._export_ply(
-            filename, bounds, color_field, color_map, color_log, sample_type
+            filename,
+            bounds,
+            color_field,
+            color_map,
+            color_log,
+            sample_type,
+            color_field_max=color_field_max,
+            color_field_min=color_field_min
         )
 
-    def _color_samples(self, cs, color_log, color_map, arr):
+    def _color_samples(self, cs, color_log, color_map, color_field_max, color_field_min, arr):
```

Comment:
Sorry I missed that one in my first round. I would suggest to put the new arguments at the end of the signature here too (ideally making them keyword only, but that's secondary)